Working good for 100M but failing for our 5M CI check. Not sure what's the best way going forward now.

```bash
_ test_xcall[fork_Prague-benchmark-gas-value_1M-blockchain_test_engine_x-opcode_CALLCODE] _
[gw4] linux -- Python 3.11.14 /opt/actions-runner/_work/execution-specs/execution-specs/.tox/benchmark/bin/python3
tests/benchmark/compute/instruction/test_system.py:102: in test_xcall
    setup_txs = math.ceil(num_contracts / num_contracts_per_tx)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E   ZeroDivisionError: division by zero
```

--- a/tests/benchmark/compute/instruction/test_system.py
+++ b/tests/benchmark/compute/instruction/test_system.py
@@ -55,7 +55,6 @@ def test_xcall(
     pre: Alloc,
     fork: Fork,
     opcode: Op,
-    env: Environment,
     gas_benchmark_value: int,
     tx_gas_limit: int,
 ) -> None:
@@ -99,6 +98,8 @@ def test_xcall(
         num_contracts_per_tx = int(tx_gas_limit * 0.9) // (
             gas_costs.G_CODE_DEPOSIT_BYTE * max_contract_size
         )
+        if num_contracts_per_tx == 0:
+            pytest.skip("tx_gas_limit too low to deploy max-size contract")
         setup_txs = math.ceil(num_contracts / num_contracts_per_tx)
 
         contracts_deployment_txs = []

